src/main/resources/jenkins/plugins/http_request/HttpRequestStep/help.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <p>
+        Performs an HTTP request, and returns a response object.
+        <p>Usage example:</p>
+        <pre>
+        def response = httpRequest url:'http://localhost:8080/jenkins/api/json?pretty=true'
+        println("Status: "+response.getStatus())
+        println("Content: "+response.getContent())
+        </pre>
+        <p>If Jenkins restarts after the HTTP request is made, but before the HTTP response is received, the build will fail.</p>
```

Comment:
Or at least the step will. As far as I know [this error](https://github.com/jenkinsci/workflow-plugin/blob/2f0724bda17ac69f033365b3fe5ac32f96a13e21/step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/AbstractSynchronousNonBlockingStepExecution.java#L72) is recoverable for example with a `try` block or the `retry` step. (untested)